.gitignore

```diff
@@ -10,3 +10,4 @@ rs.cred
 *.h5
 *.dat
 tests/run_inputs.py
+cyclus.sqlite
```

Comment:
It is sort of related because running the tests generates databases that then git will tell you are untracked.  So when running the tests here, I didn't want to accidentally commit a db.